src/dso_api/dynamic_api/remote/views.py

```diff
@@ -90,7 +92,41 @@ def list(self, request, *args, **kwargs):
         # TODO: add pagination:
         # paginator = self.pagination_class()
         # paginator.get_paginated_response(serializer.data)
-        return Response(serializer.data)
+
+        # Work around the serializer producing the wrong format.
+        # TODO: fix the serializer instead? What we get from the Kadaster remote
+        #  looks more what we want than what comes out of the serializer.
```

Comment:
indeed. Let's fix the serializer instead.

Reply:
Not in this PR, though. I was working on a different issue and just need to get this out of the way.

Reply:
Any other comments? I'd like to get on with what I was actually doing.